.github/workflows/integration-tests.yaml

```diff
@@ -352,9 +352,9 @@ jobs:
           echo "Starting Registry App (In Memory)"
           docker run -it -p 8181:8080 -e apicurio.rest.deletion.artifact.enabled=true -d ttl.sh/${{ github.sha }}/apicurio/apicurio-registry:1d
           cd registry-v2
-          ./mvnw -T 1.5C -Pintegration-tests clean install -DskipTests=true -DskipUiBuild=true -Dmaven.javadoc.skip=true
```

Comment:
Also changing from install to package since it makes very little sense to install things in the local repository here.